scripts/autoplacement_96_well.py

```diff
@@ -8,7 +8,7 @@ class Autoplacement96well(Autoplacement):
 
     output_plate_layout_rows = ["A", "B", "C", "D", "E", "F", "G", "H"]
     output_plate_layout_columns = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12"]
-    input_plate_layout_columns = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "11", "12"]
+    input_plate_layout_columns = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12"]
```

Comment:
A note for later but `input_plate_layout_columns` can be written as 
```python
input_plate_layout_columns = [str(a) for a in range(1, 13)] 
```
and     `output_plate_layout_rows`  could be written like:
```python
output_plate_layout_rows = 'ABCDEFGH'
```

Reply:
ok